### DISCUSSION TITLE
rocketmq 4.3.2 升级到 4.9.7（2m 0s)，store 位置没有改变,启动之后发现消息丢失了

### DISCUSSION THREAD

**Discussion by wangmsheng** [apache/rocketmq#7088] [Q&A]
rocketmq 4.3.2 升级到 4.9.7（2m 0s)，store 位置没有改变,启动之后发现 消息丢失了，历史消息查不到，brokerOffset 重置为0，diffTotal 变为负数了，这是什么原因

**Comment by onejimmyboy** [comment]
有没有可能是文件名改变了，之前不知道在哪的地方看到的，不同版本文件名变了
